src/index.html

```diff
@@ -117,7 +118,7 @@ <h4 class="panel-title">
                   <div class="checkbox">
                     <label class="control-label" for="compatibilityWeechat28">
                       <input type="checkbox" id="compatibilityWeechat28" ng-model="settings.compatibilityWeechat28">
-                      Compatibility with Weechat 2.8 and older <a href="#plaintext" ng-click="toggleAccordionByName('gettingStartedAccordion')"><i class="glyphicon glyphicon-info-sign"></i></a>
+                      Compatibility with Weechat 2.8 and older (Password in Plaintext) <a href="#plaintext" ng-click="toggleAccordionByName('gettingStartedAccordion')"><i class="glyphicon glyphicon-info-sign"></i></a>
```

Comment:
I find this confusing - when using TLS, the password is encrypted after all. It's being sent to the server, sure, but that's no different from every website login.

Reply:
True, I got carried away and seemed to have forgotten the possibility of using HTTPS + Pre 2.9.
I've removed the plaintext hint and also removed the warning panel, it does not really offer additional benefit compared to the existing "show tls warning" panel.